src/flinkSql/flinkStatementManager.test.ts

```diff
@@ -101,6 +102,35 @@ describe("flinkStatementManager.ts", () => {
       sandbox.restore();
     });
 
+    describe("constructor", () => {
+      it("calls setEventListeners() / sets up event handlers", () => {
+        const bogusDisposable: vscode.Disposable = {
+          dispose: () => {},
+        };
+        const fakeEventDisposables = [bogusDisposable, bogusDisposable];
+        const setEventListenersStub = sandbox
+          .stub(
+            // bending over backwards to stub private method.
```

Comment:
philosophy question: how to weigh the awkwardness of stubbing private methods, with that a method's need to be private? I get that in this case, we want events to be encapsulated to avoid race conditions etc, so just needing to stub wouldn't be enough. what are some other examples I wonder